pyproject.toml

```diff
@@ -6,16 +6,8 @@ authors = ["adrianvillanueva997"]
 
 [tool.poetry.dependencies]
 python = "3.9"
```

Comment:
Mmm, is Python needed as a dev dependency if you want to use pre-commit? As far as I know, pre-commit is just installed with pip, which short of implies the dependency to python (regardless of version). I don't think it hurts that much having it there, but I wonder about the necessity 🤔